Apps/WebClient/src/ClientApp/app/store/modules/auth/mutations.ts

```diff
@@ -0,0 +1,31 @@
+import { MutationTree } from 'vuex';
+import { AuthState, Authentication } from '@/models/authState';
+import { stat } from 'fs';
+
+export const mutations: MutationTree<AuthState> = {
+    authenticationRequest(state: AuthState) {
+        state.error = false;
+        state.statusMessage = 'loading';
+        state.requestedRoute = '';
```

Comment:
Right now the authentication request will set the requestedRoute to empty. Instead what if we pass the requested route as a parameter to the authenticationRequest mutation?
```
authenticationRequest(state: AuthState, requestedRpite: string) {
      ...
}
```

Reply:
yes, we could do that in one step for sure.